src/node_contextify.cc

```diff
@@ -934,6 +934,30 @@ Maybe<bool> StoreCodeCacheResult(
   return Just(true);
 }
 
+// TODO(RaisinTen): Reuse in ContextifyContext::CompileFunction().
+MaybeLocal<Function> CompileFunction(Isolate* isolate,
+                                     Local<Context> context,
+                                     Local<String> filename,
+                                     Local<String> content,
+                                     std::vector<Local<String>> parameters) {
```

Comment:
Why don't we pass the reference to avoid copying?

Reply:
The vectors were actually being `std::move`-ed in all the call-sites, so no copying was taking place. However, since the function doesn't really take ownership of the object, it makes sense to pass it by a pointer (not a ref because it expects it to be mutable). Done!